src/components/interfaces/HMI_API.xml

```diff
@@ -5409,6 +5433,9 @@
       <description>Image to be displayed for representing the command. See Image.</description>
       <description>If omitted, no (or the default if applicable) icon should be displayed.</description>
     </param>
+    <param name="secondaryImage" type="Common.Image" mandatory="false">
```

Comment:
It seems correct not to add `secondaryText` and `tertiaryText` here (since those field are a part of the `menuParams`) but it doesn't match https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0267-addcommand-ui-updates.md#addcommand-1. Does this require a proposal revision?

Reply:
Ah, I see there was a proposal revision https://github.com/smartdevicelink/sdl_evolution/pull/1111. Marking the comment as resolved